### PR TITLE
Add Coffee Bean & Tea Leaf to stores

### PR DESCRIPTION
**Pull request by selectfromwhere**
just add a store.

### CODE REVIEW

File: src/data/stores/theCoffeeBeanAndTeaLeaf.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  hours: [
+    [["8:00","21:00"]],
+    [["7:00","22:00"]],
+    [["7:00","22:00"]],
+    [["7:00","22:00"]],
+    [["7:00","22:00"]],
+    [["7:00","22:00"]],
+    [["7:00","22:00"]]
+  ],
+  hoursNote: 祝日 8:00 - 21:00,
```
Comment (khirayama) [inline src/data/stores/theCoffeeBeanAndTeaLeaf.ts]: I forget to add `""` for `hoursNote` in generator. Please update it to `"祝日 8:00 - 21:00"` 👍

Comment (khirayama) [reply src/data/stores/theCoffeeBeanAndTeaLeaf.ts]: @selectfromwhere Thank you for your PR.
I fixed generator by this PR https://github.com/khirayama/tmp/pull/2
Please check it 👍

File: src/data/stores/theCoffeeBeanAndTeaLeaf.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    ja: "祝日 8:00 - 21:00",
+    en: "Holiday 8:00 - 21:00",
+  },
+  email: null,
+  tel: '0362621336',
+  permanentClosed: false,
+  transforTo: null,
+  media: {
+    web: 'http://www.coffeebean.co.jp/index.html',
+    ec: 'null',
```
Comment (khirayama) [inline src/data/stores/theCoffeeBeanAndTeaLeaf.ts]: Sorry, It's my fault.
Please change `'null'` to `null` (removing single quote `'`).

File: src/data/stores/theCoffeeBeanAndTeaLeaf.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    facebook: 'https://www.facebook.com/thecoffeebean.japan',
+    twitter: 'null',
```
Comment (khirayama) [inline src/data/stores/theCoffeeBeanAndTeaLeaf.ts]: same

File: src/data/stores/theCoffeeBeanAndTeaLeaf.ts (hunk 2 above)
Comment (selectfromwhere) [reply src/data/stores/theCoffeeBeanAndTeaLeaf.ts]: Okay, I should have had a conversation before PR.
Thank you.

**Comment by khirayama** [comment]
@selectfromwhere LGTM 👍

**Comment by selectfromwhere** [comment]
Thank you!

**Pull request merged**